helmfile/charts/prometheus-alerts/templates/rules/falco-alerts.yaml

```diff
@@ -0,0 +1,31 @@
+# Generated from 'kubernetes-resources' group from https://raw.githubusercontent.com/coreos/kube-prometheus/master/manifests/prometheus-rules.yaml
+# Do not change in-place! In order to change this file first read following link:
+# https://github.com/helm/charts/tree/master/stable/prometheus-alerts/hack
+{{- if and .Values.defaultRules.create .Values.defaultRules.rules.kubernetesResources }}
```

Comment:
Sorry for not making this clear the first time. You need to add this new value [to the values](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/charts/prometheus-alerts/values.yaml#L15-L43). So just like you see `kubernetesResources: true` there, we need `falcoAlerts: true`. This is just to set the default value, otherwise it would be undefined and helm would complain about it.

Reply:
Done! :+1: